src/frontend/get-jira-configuration.ts

```diff
@@ -3,29 +3,16 @@ import moment from "moment";
 import { Subscription } from "../models";
 import { NextFunction, Request, Response } from "express";
 import statsd from "../config/statsd";
-import { metricSyncStatus, metricError } from "../config/metric-names";
-import * as Sentry from "@sentry/node";
+import { metricError } from "../config/metric-names";
 
-const syncStatus = (syncStatus) =>
-	syncStatus === "ACTIVE" ? "IN PROGRESS" : syncStatus;
-
-const sendFailedStatusMetrics = (installationId: string, reqLog: any): void => {
```

Comment:
This method would get called every time someone viewed the configuration page when the subscription had its `updatedAt` field older than 15 minutes. 

This means we were logging a status change on the read path. And the status wasn't even persisted! This is wrong on so many levels...